src/app/admin/send-email/page.tsx

```
@@ -0,0 +1,238 @@
+                <option value="late-1">Late Submission</option>
+              </select>
+            </div>
+
+            {/* Email Preview */}
+            <div className="border rounded-lg overflow-hidden">
+              <div className="bg-gray-50 p-4 border-b">
+                <h3 className="font-medium text-gray-700">Email Preview</h3>
+                <p className="text-sm text-gray-500">Subject: {getEmailSubject(selectedType)}</p>
+              </div>
+              <div className="p-4 prose max-w-none">
+                <div dangerouslySetInnerHTML={{ __html: getPreviewContent() }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)